subsys/usb/class/bluetooth.c

```diff
@@ -33,8 +33,7 @@ static K_FIFO_DEFINE(tx_queue);
 #define BLUETOOTH_IN_EP_ADDR		0x82
 
 /* TODO: Replace use of USB_MAX_FS_INT_MPS if higher speeds are supported */
-#define BLUETOOTH_BULK_EP_MPS           MIN(BT_BUF_ACL_SIZE, \
-					    USB_MAX_FS_BULK_MPS)
+#define BLUETOOTH_BULK_EP_MPS           USB_MAX_FS_BULK_MPS
 #define BLUETOOTH_INT_EP_MPS            MIN(BT_BUF_RX_SIZE, USB_MAX_FS_INT_MPS)
```

Comment:
At this point I think We can also remove this `MIN()` and leave `USB_MAX_FS_INT_MPS` as well. Going further, I would suggest deleting `BLUETOOTH_INT_EP_MPS` and `BLUETOOTH_BULK_EP_MPS` defines and use `USB_MAX_FS_BULK_MPS` and `USB_MAX_FS_INT_MPS` instead. No reason to multiply defines.

Reply:
I agree for the last part, but I think we should keep the min() for the INT EP since in the nRF connect SDK for example, BT_BUF_RX_SIZE ends up being 76 or 77 which is more than the allowed INT size (64).

Reply:
It is more and `BLUETOOTH_INT_EP_MPS` will always be 64, so why do we keep this min? I don't understand. Ep size does not determine the maximum length of a single transfer, the transfer could be bigger (>64B)

Reply:
I agree we should remove it then, in the host Kconfig, the min possible size is 73.